packages/preferences/src/browser/index.ts

```diff
@@ -15,8 +15,8 @@
  ********************************************************************************/
 
 export * from '@theia/core/lib/browser/preferences';
-export * from './abstract-resource-preference-provider';
-export * from './user-preference-provider';
-export * from './workspace-preference-provider';
-export * from './folders-preferences-provider';
-export * from './folder-preference-provider';
+export * from './providers/abstract-resource-preference-provider';
```

Comment:
please move all providers back, it is not necessary and we have other PRs which will merge conflicts with it